dtreeviz/models/tensorflow_decision_tree.py

```diff
@@ -152,8 +152,18 @@ def get_node_feature(self, id) -> int:
     def get_node_nsamples_by_class(self, id):
         all_nodes = self.internal + self.leaves
         if self.is_classifier():
-            node_value = [node.n_sample_classes() for node in all_nodes if node.id == id]
-            return node_value[0][0], node_value[0][1]
+            node = [node for node in all_nodes if node.id == id][0]
```

Comment:
yay! thanks for fix. I'll pull this in and then work on making it O(n) not O(n^2).